Run datadog formula for all hosts

bdero opened the pull request.

blarghmatey commented on salt/top.sls:
```diff
@@ -1,5 +1,7 @@
 base:
-  '*': []
+  '*':
+    - match: grain
```
The match parameter is not needed here since the match is being done with globbing.

bdero commented:
@blarghmatey Oops, I accidentally merged before you gave a thumbs up. I addressed your comment - If you have other suggestions I can either revert this or fix in a new PR.

blarghmatey commented:
:+1:
